tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -6,6 +6,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
     <LangVersion>default</LangVersion>
+    <NuGetAudit>false</NuGetAudit>
```

Comment:
Should this be added to the props for all of the projects in the integration tests directory? If so I think we already have a props file that we use for all of our tests.

Reply:
Yeah, I should have done that.  Let me push an update...